### ISSUE TITLE
UI Crate

### ISSUE THREAD

**Issue by Lowentwickler** [lowenware/dotrix#24]
* [ ] Establish a new crate in `crates/dotrix_ui`

**Comment by Lowentwickler** [comment]
Blocked by #33 and #34

**Comment by Lowentwickler** [comment]
Postponed, because the egui does the job for now and there are other important things to do.
